feat: Add support for `chunks`

bebraw opened the pull request:
This feature is similar to the one in *html-webpack-plugin* and it's required for multi-page setups.

sapegin commented on README.md:
```diff
@@ -30,12 +30,40 @@ const config = {
 
+### Multiple pages
+
+In the example below, we generate two separate pages and choose bundles from specific entries to them. This is useful for multi-page apps or sites.
```
Can we reword the first sentence to make it more clear? The "choose bundles from specific entries to them" piece doesn't make sense to me.

bebraw replied:
Yeah, I'll think of something.

sapegin commented on __snapshots__/test.js.snap:
```diff
@@ -1,5 +1,19 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`custom chunks 1`] = `
```
How can we know from this snapshot that it works correctly?

bebraw replied:
Without the `chunks` filter, it will generate a `script` tag against all entries. I'll have a look at the test, though, to see if we can simplify or state this better.

bebraw commented on test.js:
```diff
@@ -16,6 +16,30 @@ test('default options', () => {
+			],
+		}
+	).then(result => {
+		expect(result.compilation.assets['index.html']._value).toMatchSnapshot();
```
Let me add another assert here against `another.html`.

bebraw commented:
I addressed the comments and improved tests a bit so it's clear what it's doing.

Effectively we can use `MiniHtmlWebpackPlugin` as the basic building block for building sites with webpack while leveraging its bundle splitting across different pages. We match `HtmlWebpackPlugin` this way.

sapegin commented:
:tada: This PR is included in version 2.1.0 :tada:

The release is available on:
- [npm package (@latest dist-tag)](https://www.npmjs.com/package/mini-html-webpack-plugin)
- [GitHub release](https://github.com/styleguidist/mini-html-webpack-plugin/releases/tag/v2.1.0)

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:
